eolearn/visualization/eoexecutor.py

```diff
@@ -223,3 +236,21 @@ def _format_datetime(value: dt.datetime) -> str:
     def _format_timedelta(value1: dt.datetime, value2: dt.datetime) -> str:
         """Method for formatting time delta into report"""
         return str(value2 - value1)
+
+
+@dataclass()
+class _ErrorSummary:
+    """Contains data for errors of a node."""
+
+    origin: str
+    example_message: str
```

Comment:
The name of this field isn't intuitive to me. I guess you named it like this because it represents one of the multiple same error messages? why not just `error_message`?

Reply:
the error messages are not necessarily the same

Imagine specifying the folder wrong when loading eopatches. You would get 45 error messages generated by `f"Cannot find BBOX at {path}."` and since the paths would be unique, so would the error messages. This is the reason why we're now grouping by origin and not message.